views/containers/Character.jsx

```diff
@@ -10,7 +9,7 @@ const THREE = require('three');
 const Object3D = ReactTHREE.Object3D;
 const Mesh = ReactTHREE.Mesh;
 
-const keys = [KEYS.W, KEYS.A, KEYS.S, KEYS.D];
+const keys = [KEYS.Q, KEYS.W, KEYS.E, KEYS.R, KEYS.T, KEYS.ONE, KEYS.TWO, KEYS.THREE, KEYS.FOUR, KEYS.FIVE];
```

Comment:
We need to create a separate component for all the controllers, so we have only one place with all user interactions and we dispatch the related actions there. Because, this way you're coupling to much the characters with their movements.